src/webpackWorker.js

```diff
@@ -139,7 +139,7 @@ module.exports = function(configuratorFileName, options, index, expectedConfigLe
         }
         var compiler = webpack(webpackConfig);
         if(watch || webpack.watch) {
-            watcher = compiler.watch({}, finishedCallback);
+            watcher = compiler.watch(webpackConfig.watchOptions, finishedCallback);
```

Comment:
It'll default to `{}` in any case, see https://github.com/webpack/webpack/blob/v3.3.0/lib/Compiler.js#L24